### PR TITLE
Updated to downloads server, bumped version and added auth token to vali...

### PR DESCRIPTION
**Pull request by DoriftoShoes**
...dation

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -48,7 +48,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.network "private_network", ip: "172.168.90.50"
 
     # Start shell provisioning
-    config.vm.provision :shell, :inline => "curl -sS -k -O https://ops.stackstorm.net/releases/st2/scripts/st2_deploy.sh"
+    config.vm.provision :shell, :inline => "curl -sS -k -O https://downloads.stackstorm.net/releases/st2/scripts/st2_deploy.sh"
     config.vm.provision :shell, :inline => "INSTALL_WEBUI=#{webui} bash -c '. st2_deploy.sh #{st2ver}'"
```
Comment (lakshmi-kannan) [inline vagrant/Vagrantfile]: Are we going to CNAME ops.stackstorm.net to downloads.stackstorm.net?

**Comment by lakshmi-kannan** [comment]
LGTM

File: vagrant/Vagrantfile (hunk 1 above)
Comment (DoriftoShoes) [inline vagrant/Vagrantfile]: Eventually, yes.  I'm doing this in stages :)

File: vagrant/validate.sh [hunk 2]
```diff
@@ -2,7 +2,8 @@ echo "========= Verifying St2 ========="
 sleep 10
 st2ctl status
 echo "========== Test Action =========="
-st2 run core.local hostname
+TOKEN=`st2 auth ${TEST_ACCOUNT_USERNAME} -p ${TEST_ACCOUNT_PASSWORD} | grep token | awk '{print $4}'`
```
Comment (Kami) [inline vagrant/validate.sh]: This command might fail (exit with non zero) in versions pre 0.9dev where auth is not running by default, but I don't see us do `set -e` in this script so I think we should be fine.

(just thinking out loud)

Comment (DoriftoShoes) [inline vagrant/validate.sh]: That's actually fine.  Someone would have to go out of their way to download the 0.9dev st2_deploy.sh and then run it to install an earlier version.  The wrapper script grabs the st2_deploy for the version they are trying to install...and 0.8 doesn't have the auth foo as part of the deployment.

Comment (Kami) [inline vagrant/validate.sh]: Oh right, good point.

**Pull request merged**